fedora_messaging/tests/unit/test_callbacks.py

```diff
@@ -48,6 +49,18 @@ def test_printer(self):
         )
 
         with mock.patch("sys.stdout", new_callable=StringIO) as mock_stdout:
-            example.printer(message)
+            callbacks.printer(message)
 
         self.assertEqual(expected_stdout, mock_stdout.getvalue())
+
+
+class RunnerTests(unittest.TestCase):
+    def test_runner(self):
+        """ Assert the run callback calls a subprocess."""
+        config.conf = config.LazyConfig()
+        config.conf["consumer_config"]["command"] = "/bin/true"
+        message = api.Message(body="Hello world", topic="hi")
+        with mock.patch("subprocess.check_call") as mock_subprocess:
+            callbacks.run(message)
+
+        mock_subprocess.assert_called_once_with("/bin/true", shell=False, timeout=None)
```

Comment:
Could you also check that the `Drop` exception is raised when the command fails, and that the proper exception is raised when the command times out?